selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/miscellaneous/FindTextFeatureTest.java

```diff
@@ -237,8 +242,12 @@ public void checkFindTextBasic() {
     findText.clickOnSearchButtonMainForm();
     findText.waitFindInfoPanelIsOpen();
     findText.waitExpectedTextInFindInfoPanel(FIND_NOTHING);
-
-    findText.clickHideBtnFindInfoPanel();
+    consoles.clickOnProcessesButton();
```

Comment:
let's add empty rows between the test steps to improve readability.

Reply:
Have done